src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/Diff/DiskDiffEntry.cs

```diff
@@ -0,0 +1,31 @@
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Hashing.xxHash64;
+using NexusMods.Paths;
+
+namespace NexusMods.Abstractions.Loadouts.Synchronizers;
+
+/// <summary>
+/// A file diff entry
+/// </summary>
+public record struct DiskDiffEntry
```

Comment:
Consider setting this to `Record Class` for the time being.

I generally consider the threshold of struct profitability to be 4 registers. This has 6 variables (2 in Size), (2 in GamePath), thus is considerably larger.

The one in `Size` should probably be patched out by switching away from Vogen in paths, but GamePath will always be 2 vars.